src/fparser/two/utils.py

```diff
@@ -650,6 +669,8 @@ def tofortran(self, tab='', isfix=None):
             mylist.append(item.tofortran(tab=tab+extra_tab, isfix=isfix))
         if len(self.content) > 1:
             mylist.append(end.tofortran(tab=tab, isfix=isfix))
+        # Ensure all strings in list are encoded consistently
+        py2_encode_list_items(mylist)
```

Comment:
Is there a test for this?

Reply:
Yes - it was the one that I've just said I've removed. I've put it back and made it call this method explicitly.